config/local-test.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Please remove this file from the git. You probably first added it, then modified .gitignore – that's why it didn't work.